### ISSUE TITLE
sbt-allsources.sh fails with `no project 'compilerBridge'` in zinc

### ISSUE THREAD

**Issue by muuki88** [sbt/sbt#4330]
- [x] I've read the [CONTRIBUTING](https://github.com/sbt/sbt/blob/1.x/CONTRIBUTING.md) guidelines

## steps

Run as described in the [CONTRIBUTING](https://github.com/sbt/sbt/blob/1.x/CONTRIBUTING.md)

```bash
$ mkdir sbt-modules
$ cd sbt-modules
$ for i in sbt io util librarymanagement zinc; do \
  git clone https://github.com/sbt/$i.git && (cd $i; git checkout -b 1.x origin/1.x)
done
$ cd sbt
$ ./sbt-allsources.sh
```

## problem

```
java.lang.RuntimeException: No project 'compilerBridge' in 'file:/home/muki/dev/git/github/sbt/zinc/'.
```

## expectation

No runtime exceptions

## notes

It seems that [this commit](https://github.com/sbt/zinc/commit/5b179a2611c48ac928239d1ec003b536e5c5e02a#diff-fdc3abdfd754eeb24090dbd90aeec2ce) is responsible for this issue.

**Comment by nhooey** [comment]
@eed3si9n The fix for this never made it in to the `1.2.x` branch.

**Comment by nhooey** [comment]
@eed3si9n Sorry that's not true, I found the fix, but I'm having [another related issue](https://stackoverflow.com/questions/52461497).
